src/components/Tabs/__tests__/Tabs.spec.js

```diff
@@ -35,5 +37,28 @@ describe('<Tabs> that renders tab container with some tabs', () => {
             .at(1)
             .simulate('click');
         expect(wrapper.state().activeTabId).toBe('2');
+        expect(onChange).toHaveBeenCalledWith('2');
+    });
+
+    it('should switch tabs correctly (programmatically)', () => {
+        const wrapper = mount(
+            <Tabs activeTabId="1">
+                <Tab id="1" label="First">
+                    Some content
+                </Tab>
+                <Tab id="2" label="Second">
+                    Some more content
+                </Tab>
+            </Tabs>
+        );
+        wrapper.setProps(
+            {
+                activeTabId: '2'
+            },
+            () => {
+                expect(wrapper.state().activeTabId).toBe('2');
+                expect(wrapper.state().derivedTabId).toBe('2');
```

Comment:
Please add a check to see that the correct content is displayed. Maybe a snapshot test, or something more explicit,  like adding a paragraph to the content that you can search the wrapper for.

Reply:
Good idea, done